config/environments/development.rb

```diff
@@ -1,6 +1,9 @@
 require "dotenv/load"
 
 Rails.application.configure do
+  # Prepare the ingress controller used to receive mail
+  config.action_mailbox.ingress = :sendgrid
```

Comment:
Should we be using SendGrid in development? Answer may be yes - just wanted to make sure.

Reply:
the ingress can be configured without it actually bringing in emails. I think this is optional, but doesn't hurt and allows for easier testing locally.